io.github.yesser_studios.Pong.yml

```diff
@@ -0,0 +1,51 @@
+id: io.github.yesser_studios.Pong
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+finish-args:
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=host-os:ro
+
+command: run.sh
+modules:
+  - name: dotnet
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/dotnet8/bin/install.sh
+
+  - name: pong
+    buildsystem: simple
+    build-options:
+      prepend-path: "/usr/lib/sdk/dotnet8/bin"
+      append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+      prepend-pkg-config-path: "/usr/lib/sdk/dotnet8/lib/pkgconfig"
+      arch:
+        aarch64:
+          env:
+            RUNTIME: linux-arm64
+        x86_64:
+          env:
+            RUNTIME: linux-x64
+    build-commands:
+      - cp -r --remove-destination ./bin/* ${FLATPAK_DEST}/bin/
+      - install -Dm644 flatpak/io.github.yesser_studios.Pong.desktop -t ${FLATPAK_DEST}/share/applications/
+      - install -Dm644 flatpak/io.github.yesser_studios.Pong.png -t ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps
+      - install -Dm644 flatpak/io.github.yesser_studios.Pong.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo
```

Comment:
I don't see where these three files exist at that point in time.

Reply:
did you try to build this locally?

Reply:
Yes, these files exist in the project's repository: https://github.com/yesser-studios/Pong/tree/main/flatpak

Reply:
ah right there is the git checkout